.github/workflows/windows_workflow.yml

```
@@ -0,0 +1,36 @@
+name: windows-workflow
+
+on:
+  push:
+    branches: 
+      - master
+  pull_request:
+    branches:
+      - '*'
+    paths:
+      - '**'
+      - '!flank-scripts/**'
+
+jobs:
+  build:
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v2
+        with: 
+          submodules: true
+      - name: Display the java version
+        run: java -version 
+        shell: bash
+
+      - uses: actions/cache@v2
+        with:
+          path: ~/.gradle/caches
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+
+      - name: Gradle clean build
+        uses: eskatos/gradle-command-action@v1
+        with:
+          gradle-executable: "./test_runner/gradlew"
+          arguments: "-p test_runner clean build"
```

Now we should use only arguments without executable
```suggestion
          arguments: ":test_runner:clean :test_runner:build"
```

---

please check ubuntu or macos for reference